README.md

```diff
@@ -4,16 +4,67 @@
 
 ## Usage
 
+Formulator is a library to give you:
+* A label for your input, based on the attribute name
+* An error label
+* A class around the input if there is an error for the attribute.
+
+You can replace the following:
+```elixir
+  <%= label form, :email %>
+  <%= text_input form, :email %>
+  <%= error_tag form, :email %>
+```
+
+with the following:
```

Comment:
I think "with this" would read a little better (just because of the repeated "the following").